streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -422,8 +426,11 @@ private void closeOpenIterators() {
         synchronized (openIterators) {
             iterators = new HashSet<>(openIterators);
         }
-        for (final KeyValueIterator iterator : iterators) {
-            iterator.close();
+        if (iterators.size() != 0) {
+            log.warn("Closing {} open iterators for store {}", iterators.size(), name);
```

Comment:
Why do we want to log it as WARN?

Reply:
For our own code base, this should never happen. For IQ, it's a different story. I though it might be helpful to expose potential bugs in our code base? We can also log at debug level if you prefer.

Reply:
Maybe we could put this to `INFO` but change the message slightly to say "This could be an issue for IQ" or something along those lines.  Just a thought.

Reply:
I think @guozhangwang would prefer less verbose log level like DEBUG? Not sure.

Like the idea to mention IQ explicitly.

Reply:
I don't have a strong opinion on the log level `DEBUG` is fine with me.

Reply:
@mjsax I think I'm sold on your arguments, let's keep them as WARN then :)